projects/igniteui-angular/src/lib/radio/radio.component.html

```diff
@@ -9,10 +9,10 @@
     [attr.aria-checked]="checked"
     [attr.aria-labelledby]="ariaLabelledBy"
     [attr.aria-label]="ariaLabel"
-    (click)="_clicked($event)"
```

Comment:
Shouldn't the input itself still be handling click together with the label?

Reply:
The native input is hidden, hence why it doesn't make sense to listen for the click even on it.